package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "scripts": {
+    "lint-scss": "sass-lint static/**/*.scss --verbose --no-exit",
+    "build": "yarn run build-css",
+    "build-css": "node-sass --include-path node_modules static/sass --output static/css && postcss --use autoprefixer --replace 'static/css/**/*.css' && postcss --use cssnano --dir static/minified 'static/css/**/*.css'",
+    "serve": "./entrypoint 0.0.0.0:${PORT}",
+    "watch": "watch -p 'static/sass/**/*.scss' -p 'static/js/**/*.js' -c 'yarn run build'",
+    "watch-scss": "watch -p 'static/sass/**/*.scss' -c 'yarn run build-css'",
+    "clean": "rm -rf node_modules yarn-error.log css static/css *.log *.sqlite _site/ build/ .jekyll-metadata .bundle",
```

Comment:
don't know if you played, but can you remove the useless stuff to clean?
`.jekyll-metadata` or `.bundle` for ex